app/Image.php

```diff
@@ -0,0 +1,10 @@
+<?php
```

Comment:
Why, do we use this model

Reply:
when I was doing the dropzone I had some problems and I tried with the model, but after it was received without it